content/docs/2.12/concepts/scaling-deployments.md

```diff
@@ -286,17 +296,17 @@ advanced:
   scalingModifiers:
     formula: "(trig_one + trig_two)/2"
     target: "2"
+    activationTarget: "2"
```

Comment:
Could you please add activationTarget with an explanation to examples below?

Reply:
done